src/Assets/ugui-mvvm/PropertyBinding.cs

```diff
@@ -324,6 +324,20 @@ public void ClearHandlers()
         PropertyPath _vProp;
         PropertyPath _vmProp;
 
+        public void CopyFromExisting(PropertyBinding other)
```

Comment:
Is this method being used? Does this provide different behavior from [`EditorUtility.CopySerialized`](https://docs.unity3d.com/ScriptReference/EditorUtility.CopySerialized.html)?

Reply:
Cool.  Didn't know about that.  I think it's functionally equivalent, so I'll switch to it.